pinot-server/src/main/java/org/apache/pinot/server/api/resources/ControllerJobStatusResource.java

```diff
@@ -51,45 +54,32 @@ public class ControllerJobStatusResource {
   @ApiOperation(value = "Task status", notes = "Return the status of a given reload job")
   public String reloadJobStatus(@PathParam("tableNameWithType") String tableNameWithType,
       @QueryParam("reloadJobTimestamp") long reloadJobSubmissionTimestamp,
-      @QueryParam("segmentName") String segmentName,
-      @Context HttpHeaders headers)
+      @QueryParam("segmentNames") String segmentNames, @Context HttpHeaders headers)
```

Comment:
This is backward incompatible. Let's just keep the param as is

Reply:
This is a server side API and not used often so I updated it but I’m ok to keep it intact.